### ISSUE TITLE
OpenSIPS 2.1 crash when replicate_contacts_to enabled

### ISSUE THREAD

**Issue by staskobzar** [OpenSIPS/opensips#652]
Hello Sirs,

There is an issue with bin replication. I have tried on CentOS and Debian.
OpenSIPS crash with usrloc bin replication enabled (segfault):
modparam("usrloc", "replicate_contacts_to", "192.168.1.139:5062")

OpenSIPS version:
version: opensips 2.1.1 (x86_64/linux)
flags: STATS: On, DISABLE_NAGLE, USE_MCAST, SHM_MMAP, PKG_MALLOC, F_MALLOC, FAST_LOCK-ADAPTIVE_WAIT
ADAPTIVE_WAIT_LOOPS=1024, MAX_RECV_BUFFER_SIZE 262144, MAX_LISTEN 16, MAX_URI_SIZE 1024, BUF_SIZE 65535
poll method support: poll, epoll_lt, epoll_et, sigio_rt, select.
git revision: 4963c7c
main.c compiled on 20:32:30 Sep 26 2015 with gcc 4.4.7

core "bt full": https://www.dropbox.com/s/zmtkepwzanyfmq4/core.txt?dl=0
OpenSIPS config file: https://www.dropbox.com/s/ut635s8s4d9axta/opensips.cfg.txt?dl=0
OpenSIPS log: https://www.dropbox.com/s/1ohh5f03sbvy3ej/opensips.log.txt?dl=0

Best regards,
Stas Kobzar

**Comment by razvancrainea** [comment]
Hi, Stas!

You do not have any bin_listeners in your configuration file, so OpenSIPS does not know what to use to replicate the information. The crash itself is indeed a bug, and we are working on a fox for this. Meanwhile, setting a ```bin_listen=192.168.1.139:5070``` directive to your script should temporary fix your problem.

Best regards,
Răzvan

**Comment by staskobzar** [comment]
Hello Răzvan,

You are right, I do not have bin_listen. And it works perfect after I have added this parameter.

I actually wanted this OpenSIPS instance only to send bin replications, that's why I did not configured this parameter. But it is obvious that to send, OpenSIPS also needs to know which interface to use. 

So it is also my mistake. It would be great though if OpenSIPS can give a warning/error about missing this parameter in startup logs too.

Thank you,
Stas
